.github/workflows/build_pip.yaml

```
@@ -0,0 +1,58 @@
+name: Editable build using pip and pre-release NumPy
+
+on: push
+
+permissions: read-all
+
+env:
+  PACKAGE_NAME: mkl_fft
+  MODULE_NAME: mkl_fft
+  TEST_ENV_NAME: test_mkl_fft
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+         shell: bash -el {0}
+
+    strategy:
+      matrix:
+        python: ['3.9', '3.10', '3.11', '3.12']
+        use_pre: ["", "--pre"]
+
+    steps:
+      - name: Install jq
+        shell: bash -l {0}
+        run: |
+          sudo apt-get install jq
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: conda-incubator/setup-miniconda@v3
+        with:
+          use-mamba: true
+          mamba-version: "*"
```

Would it be helpful to switch to `Mambaforge` if we are going to use mamba?
```suggestion
          miniforge-variant: Mambaforge
          miniforge-version: latest
          use-mamba: true
```